game/src/main/scala/io/aigar/game/Resources.scala

```diff
@@ -48,7 +48,7 @@ class ResourceType(grid:Grid, val min: Int, val max: Int, mass: Int, score: Int)
     val ratio = (positions.length - min).toFloat / (max - min)
 
     // Add MinPositiveValue to prevent obtaining zero since ratio can be 0
-    if ((scala.util.Random.nextFloat + scala.Float.MinPositiveValue) >= ratio) {
+    if (scala.util.Random.nextFloat >= ratio) {
```

Comment:
I think this is the place where the change was removed and that's why I was confused. Was this on purpose?